.github/workflows/client-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: client-ci 
+on:
+  push:
+    branches:
+      - main
+jobs:
+  clientdocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      - run: pip install mkdocs-material
```

Comment:
you can make this all into one step:
- name: Install Dependencies
   run: pip install mkdocs-material mkdocs mkdocstrings-python .......
   
pip can install a list of packages